### PR TITLE
Various Endpoints

### PR DESCRIPTION
**Pull request by skorfmann**
Thanks for your initial effort, this got me going quickly so I added a few endpoints. It's still work in progress, though.

A few things I noted / questions:

- camel-casing the response doesn't seem to work for me 
- Had to downgrade `axios` due to [type errors](https://github.com/axios/axios/issues/3219)
- what's the best way to reference it locally? (e.g. yarn link) (Had to add `main` / `types` entry to `package.json` to make it work)

### CODE REVIEW

**Comment by mijailr** [comment]
Looks great, lets see your issues, please give me more details, I'm happy to help.

File: src/api/terraformCloudApiClient.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ const terraformCloudApiClient = (apiKey: string): AxiosInstance => {
     return req
   })
 
-  client.interceptors.response.use((res: AxiosResponse) => camelcaseKeys(res.data))
+  client.interceptors.response.use((res: AxiosResponse) => camelcaseKeys(res.data, { deep: true }))
```
Comment (skorfmann) [inline src/api/terraformCloudApiClient.ts:18]: This fixed the camel-casing for nested keys

Comment (mijailr) [reply src/api/terraformCloudApiClient.ts:18]: 🎉

**Comment by skorfmann** [comment]
The "issues" in [codeclimate](https://codeclimate.com/github/mijailr/terraform-cloud/pull/114) are a bit weird

**Comment by skorfmann** [comment]
> * what's the best way to reference it locally? (e.g. yarn link) (Had to add `main` / `types` entry to `package.json` to make it work)

I thought `semantic-release` is doing some magic there. However, looks like these entries are really missing.

**Comment by mijailr** [comment]
> The "issues" in [codeclimate](https://codeclimate.com/github/mijailr/terraform-cloud/pull/114) are a bit weird

Fixed

> > * what's the best way to reference it locally? (e.g. yarn link) (Had to add `main` / `types` entry to `package.json` to make it work)
> 
> I thought `semantic-release` is doing some magic there. However, looks like these entries are really missing.

You mean to link the terraform client as a path on another project? I never tried that before, maybe the packaging is needed. Do you know a way to fix that?

**Comment by skorfmann** [comment]
> > The "issues" in [codeclimate](https://codeclimate.com/github/mijailr/terraform-cloud/pull/114) are a bit weird
> 
> Fixed

👍 

> You mean to link the terraform client as a path on another project? I never tried that before, maybe the packaging is needed. Do you know a way to fix that?

I mean even for the published npm package (I did publish a [temporary package in npm](https://www.npmjs.com/package/@skorfmann/terraform-cloud)) . And yes, for the local workflows as well.

**Comment by mijailr** [comment]
Maybe `pika` is not the best option for the packaging in this case or we need to use pika in another way. Let me know when we can merge this.

**Comment by skorfmann** [comment]
@mijailr please have a look - that's the state what we're using at the moment. However, happy to expand on it.

**Review by mijailr — APPROVED** [review]
👍🏽 LGTM!

**Comment by mijailr** [comment]
Many thanks @skorfmann

**Pull request merged**